src/redux/data.ts

```diff
@@ -1360,7 +1359,7 @@ export const dataWatchPendingTransactions = (
         status: TransactionStatus;
       } | null = {
         status: TransactionStatus.sending,
-        title: '',
+        title: pendingTransactionData?.title || TransactionStatus.sending,
```

Comment:
this was causing empty tittles if nothing happens in the logic below

we should use the same title we has previously and if its not there default to a generic send